.github/CODEOWNERS

```diff
@@ -103,53 +103,53 @@
 /core/domain/value_generators_domain*.py @vojtechjelinek
 /core/storage/exploration/ @aks681
-/core/templates/dev/head/components/entity-creation-services/exploration-creation.service*.ts @DubeySandeep
+/core/templates/dev/head/components/entity-creation-services/exploration-creation.service*.ts @aks681
 /core/templates/dev/head/components/button-directives/hint-and-solution-buttons.directive*.ts @aks681
 /core/templates/dev/head/components/button-directives/hint-and-solution-buttons.directive.html @aks681
 /core/templates/dev/head/components/state-directives/ @aks681
-/core/templates/dev/head/components/graph-services/ @DubeySandeep
-/core/templates/dev/head/components/version-diff-visualization/version-diff-visualization.directive.html @DubeySandeep
-/core/templates/dev/head/components/version-diff-visualization/version-diff-visualization.directive.ts @DubeySandeep
-/core/templates/dev/head/domain/editor/ @DubeySandeep
-/core/templates/dev/head/domain/exploration/ @DubeySandeep
+/core/templates/dev/head/components/graph-services/ @aks681
+/core/templates/dev/head/components/version-diff-visualization/version-diff-visualization.directive.html @aks681
+/core/templates/dev/head/components/version-diff-visualization/version-diff-visualization.directive.ts @aks681
+/core/templates/dev/head/domain/editor/ @aks681
+/core/templates/dev/head/domain/exploration/ @aks681
 /core/templates/dev/head/domain/state/ @aks681
 /core/templates/dev/head/domain/summary/ @aks681
 /core/templates/dev/head/domain/state_card/ @aks681
-/core/templates/dev/head/pages/exploration-editor-page/ @DubeySandeep
+/core/templates/dev/head/pages/exploration-editor-page/ @aks681
 /core/templates/dev/head/pages/exploration-player-page/ @aks681
 /core/templates/dev/head/components/state-editor/ @aks681
-/core/templates/dev/head/services/compute-graph.service.ts @DubeySandeep
+/core/templates/dev/head/services/compute-graph.service.ts @aks681
 /core/templates/dev/head/services/context.service*.ts @aks681
-/core/templates/dev/head/services/editability.service*.ts @DubeySandeep
+/core/templates/dev/head/services/editability.service*.ts @aks681
 /core/templates/dev/head/services/exploration-features*.ts @aks681
 /core/templates/dev/head/services/exploration-html-formatter.service*.ts @aks681
-/core/templates/dev/head/services/validators.service*.ts @DubeySandeep
+/core/templates/dev/head/services/validators.service*.ts @aks681
 
 
 # Forms pages.
-/core/templates/dev/head/components/forms/ @DubeySandeep
-/core/templates/dev/head/components/common-layout-directives/common-elements/confirmation-modal.template.html @DubeySandeep
-/core/templates/dev/head/components/ck-editor-helpers/customize-rte-component-modal.template.html @DubeySandeep
-/core/templates/dev/head/services/schema-default-value.service.ts @DubeySandeep
-/core/templates/dev/head/services/schema-undefined-last-element.service.ts @DubeySandeep
+/core/templates/dev/head/components/forms/ @bansalnitish
```

Comment:
Curious -- Why I'm assigned this part?

Reply:
I have basically assigned you to the miscellaneous frontend files as you are handling the angular/frontend migration as of now and I think this will help you enforce others to do the changes in Angular8 in the first place.

Reply:
Okay, sg.